storage/kv_store_rocksdb_int_keys.py

```diff
@@ -0,0 +1,49 @@
+from storage.kv_store_rocksdb import KeyValueStorageRocksdb
+
+try:
+    import rocksdb
+except ImportError:
+    print('Cannot import rocksdb, please install')
+
+
+class IntegerComparator(rocksdb.IComparator):
+    def compare(self, a, b):
+        a = int(a)
+        b = int(b)
+        if (a < b):
+            return -1
+        if (a > b):
+            return 1
+        return 0
+
+    def name(self):
+        return b'IntegerComparator'
+
+
+class KeyValueStorageRocksdbIntKeys(KeyValueStorageRocksdb):
+    def __init__(self, db_dir, db_name, open=True):
+        super().__init__(db_dir, db_name, open)
+
+    def open(self):
+        opts = rocksdb.Options()
+        opts.create_if_missing = True
+        opts.comparator = IntegerComparator()
+        self._db = rocksdb.DB(self._db_path, opts)
+
+    def get_equal_or_prev(self, key):
```

Comment:
Why don't we implement it in KeyValueStorageRocksdb?

Reply:
Because it is correct just in case of integer keys with corresponding comparator. Otherwise the order is lexicographical where '2' is greater than '100', so get_equal_or_prev() does not make sense.